### DISCUSSION TITLE
Fluxstore Pixels for Marketing and Remarketing Facebook Ads and Google Ads

### DISCUSSION THREAD

**Discussion by jsaristi** [inspireui/support#5581] [New Feature/Ideas Requests]
Hello, I have realized that the application does not have a connection to analytics or pixels that help to do adequate marketing for the application, for example, configure it to be able to know on the different platforms such as facebook pixel or google tag manager, when someone made a purchase (to carry out CTA campaigns - Cost to Acquire) or when someone saw a product and it remarketed it on different sites or pages or social network apps

https://www.facebook.com/business/help/1044262445604547?id=1913105122334058

https://developers.google.com/tag-platform/tag-manager/ios/v5

https://developers.google.com/tag-platform/tag-manager/android/v5

**Comment by jsaristi** [comment]
Hello, Any notice from this?

**Comment by jsaristi** [comment]
This feature its easy to do, its just apply the code of the pixel for facebook and the tag for google analitycs and ads for a better perfomance of the ads and analitycs,

**Comment by osman-kocak** [comment]
up

**Comment by jsaristi** [comment]
Hello, I see that in the last update they installed Google Analytics, excellent progress.

Any update regarding this new functionality that is extremely important for marketing?
